modules/cas_cache/layer_cache_management.c

```diff
@@ -2184,6 +2260,54 @@ int cache_mngt_get_seq_cutoff_policy(ocf_core_t core,
 	return result;
 }
 
+/**
+ * @brief Get sequential cutoff promotion threshold value
+ * @param[in] core OCF core
+ * @param[out] thresh sequential cutoff promotion threshold value
+ * @return exit code of successful completion is 0;
+ * nonzero exit code means failure
+ */
+
+int cache_mngt_get_seq_cutoff_promotion_threshold(ocf_core_t core,
+		uint32_t *thresh)
+{
+	ocf_cache_t cache = ocf_core_get_cache(core);
+	int result;
+
+	result = _cache_mngt_read_lock_sync(cache);
+	if (result)
+		return result;
+
+	result = ocf_mngt_core_get_seq_cutoff_promotion_threshold(core, thresh);
+
+	ocf_mngt_cache_read_unlock(cache);
+	return result;
+}
+
+/**
+ * @brief Get sequential cutoff promotion request count value
+ * @param[in] core OCF core
+ * @param[out] thresh sequential cutoff promotion request count value
```

Comment:
Did you mean 'count' instead of 'thresh'?

Reply:
Yep. Fixed.